eth/vm/forks/berlin/transactions.py

```diff
@@ -371,15 +376,15 @@ def max_fee_per_gas(self) -> int:
 
     @property
     def max_fee_per_blob_gas(self) -> int:
-        raise NotImplementedError(
-            "max_fee_per_blob_gas is not implemented until Cancun."
-        )
+        return self._inner.max_fee_per_blob_gas
```

Comment:
Will this (and `blob_versioned_hashes`, and `authorization_list` below) cause a problem if called from the berlin vm?

Reply:
Each `TypedTransaction` implementation has their own `@property` defining the behavior. So if it's an `AccessListTransaction`, [it will cause a problem](https://github.com/ethereum/py-evm/blob/prague/eth/vm/forks/berlin/transactions.py#L280-L283) because this isn't part of that transaction.